quic/s2n-quic-transport/src/transmission/application.rs

```diff
@@ -84,10 +83,6 @@ impl<'a, Config: endpoint::Config> super::Payload for Payload<'a, Config> {
             Payload::PathValidationOnly(inner) => inner.on_transmit(context),
         }
     }
-
-    fn packet_number_space(&self) -> PacketNumberSpace {
```

Comment:
Were these just unused?

Reply:
apparently, I'm not sure why it just started popping up in clippy though

Reply:
The trait is internal to the crate and clippy just started being able to detect this type of dead code